server/routes/caseAdmin/assessment.ts

```diff
@@ -12,7 +16,95 @@ export default class AssessmentRoutes {
     )
 
     res.render('pages/caseAdmin/assessment', {
-      assessmentSummary: { ...assessmentSummary, tasks: assessmentSummary.tasks.PRISON_CA },
+      assessmentSummary: {
+        ...assessmentSummary,
+        tasks: assessmentSummary.tasks.PRISON_CA,
+        formsToShow: this.getFormsToShow(assessmentSummary),
+      },
     })
   }
+
+  private getFormsToShow(assessmentSummary: AssessmentOverviewSummary): { link: string; text: string }[] {
+    const { prisonNumber, status } = assessmentSummary
+    const baseForms = [
+      {
+        link: this.getFormPath(prisonNumber, documentSubjectType.OFFENDER_ELIGIBLE_FORM),
+        text: 'Eligible',
+      },
+      {
+        link: this.getFormPath(prisonNumber, documentSubjectType.OFFENDER_ADDRESS_CHECKS_INFORMATION_FORM),
+        text: 'Information about address checks',
+      },
+      {
+        link: this.getFormPath(prisonNumber, documentSubjectType.OFFENDER_ADDRESS_CHECKS_FORM),
+        text: 'Address checks',
+      },
+      {
+        link: this.getFormPath(prisonNumber, documentSubjectType.OFFENDER_OPT_OUT_FORM),
+        text: 'Opt out',
+      },
+    ]
+
+    return status === AssessmentStatus.INELIGIBLE_OR_UNSUITABLE || status === AssessmentStatus.NOT_STARTED
+      ? this.getStatusForms(prisonNumber, status as AssessmentStatus)
+      : [...baseForms, ...this.getStatusForms(prisonNumber, status as AssessmentStatus)]
+  }
+
+  private getStatusForms(prisonNumber: string, status: AssessmentStatus): { link: string; text: string }[] {
+    const statusFormsMap: { [key in AssessmentStatus]?: { link: string; text: string }[] } = {
+      [AssessmentStatus.ELIGIBLE_AND_SUITABLE]: [],
```

Comment:
can we move this map to `/server/config`?

making the link a function that takes a `prisonNumber`

```
   const formList = forms[status] || []
```